src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -2273,8 +2273,8 @@ public static void Main(string[] args)
                 }
                 else
                 {
-                    thread1.Start();
                     thread2.Start();
+                    thread1.Start();
```

Comment:
Read the context. The purpose of the enclosing `if` statement is to permute the order in which the threads are started.